src/main/java/frc/robot/subsystems/internals/InternalSubsystem.java

```diff
@@ -103,15 +99,14 @@ public InternalSubsystem(TurretSubsystem turretSubsystem) {
 
         // if we are in auton we start with 1 ball
         if (DriverStation.isAutonomous()) {
```

Comment:
Since this is called in the subsystem constructor (and the constructor invoked during robot initialization) I don't think this will work to set `ballCount` to 1 when autonomous initializes. We might need to put it in `Robot.autonomousInit()` and pass `InternalSubsystem` from `RobotContainer` to `Robot` via a getter.

Reply:
We can test out both ways. Can you test in shop to see if the ball count prints as 1 when we start autonomously?

Reply:
<img width="512" alt="auton" src="https://user-images.githubusercontent.com/60120929/154872437-2c5cdd8e-9391-4d43-8e0a-23fff216b66a.png">
Ball count is 0 when code deploy (robot initialization) occurs when driver station is on the teleop tab, and stays at 0 when switching to (and enabling) auton.

<img width="701" alt="auton2" src="https://user-images.githubusercontent.com/60120929/154872469-be1e1780-8833-45fc-b8bc-93922afb4c0f.png">
Ball count is 1 only when the robot initializes while the driver station is on the auton tab.

The idea involving `Robot.autonomousInit()` would set `ballCount` to 1 every time autonomous begins which is something to watch for in testing if we're running autonomous repeatedly.